spyder/app/tests/conftest.py

```diff
@@ -17,6 +17,7 @@
 from qtpy.QtCore import Qt
 from qtpy.QtTest import QTest
 from qtpy.QtWidgets import QApplication, QFileDialog, QLineEdit, QTabBar
+from qtpy import QtWebEngineWidgets
```

Comment:
```suggestion
# This is required to run our tests in VSCode or Spyder-unittest
from qtpy import QtWebEngineWidgets  # noqa
```

Add a clarifying comment and `# noqa` so we don't remove this line in the future if it's reported by linters as unnecessary.